File: csharp/VS2005/netframework/Modules/25.Printing and Exporting/20.CustomPreview/PasswordForm.cs
using System;
using System.Drawing;
using System.ComponentModel;
using System.Windows.Forms;

namespace CustomPreview
{
    /// <summary>
    /// Form for asking for a password when the file is password protected.
    /// </summary>
    public partial class PasswordForm: System.Windows.Forms.Form
    {

        public PasswordForm()
        {
            InitializeComponent();
        }

        public string Password
        {
            get
            {
                return PasswordEdit.Text;
            }
        }
    }
}

File: vb/VS2022/netframework/Modules/25.Printing and Exporting/20.CustomPreview/PasswordForm.vb
Imports System.ComponentModel

Namespace CustomPreview
	''' <summary>
	''' Form for asking for a password when the file is password protected.
	''' </summary>
	Partial Public Class PasswordForm
		Inherits System.Windows.Forms.Form

		Public Sub New()
			InitializeComponent()
		End Sub

		Public ReadOnly Property Password() As String
			Get
				Return PasswordEdit.Text
			End Get
		End Property
	End Class
End Namespace
